iOS/UI/Updates/MangaUpdatesView.swift

```diff
@@ -92,17 +92,23 @@ struct MangaUpdatesView: View {
                 let items = entry.items
                 ForEach(items, id: \.mangaKey) { item in
                     let updates = item.updates
-                    if let manga = updates.first?.manga {
+                    if let update = updates.first {
                         NavigationLink(
-                            destination: MangaView(manga: manga, path: path)
+                            destination: MangaView(manga: update.manga, path: path)
                                 .onAppear {
-                                    setOpened(manga: manga)
+                                    setOpened(manga: update.manga)
                                 }
                         ) {
                             MangaUpdateItemView(updates: updates)
                         }
                         .offsetListSeparator()
                         .id(item.mangaKey)
+                        .swipeActions(edge: .trailing) {
+                            Button(NSLocalizedString("DELETE")) {
+                                removeUpdate(update: update, day: entry.day)
+                            }
+                            .tint(.red)
+                        }
```

Comment:
can we use `onDelete` instead of `swipeActions` for the system styling? if not, we should copy the style in `BackupsView` and use a destructive button with a `Label` with the trash icon.